parsons/utilities/check_env.py

```diff
@@ -1,9 +1,13 @@
 import os
 
 
-def check(env, field=None):
-    """Check if an envrionment variable is set for __init__"""
-    if not field:
+def check(env, field, *args):
+    """
+    Check if an environment variable has been set. If it has not been set
+    and the passed field or arguments have not been passed, then raise an
+    error.
+    """
+    if not field and not args:
```

Comment:
Above, you are passing in `pgpass` and I think you are trying to skip the `os.environ` check if `pgpass` is `True`. But`not args` will always be `True` if you pass in any additional args, even if the value is `False`.

Reply:
It was intentional, though admittedly a little hacky.

Reply:
So, I think that even if the pgpass file doesn't exist, `not args` will be False, and so you will never go to the environment. Because if you pass in any additional arguments to `check`, `args` is truthy. Could you just add a new optional positional argument that is `skip` or something that means "just give me `field` back"?